lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -11,6 +11,14 @@ const Js::OpCode LowererMD::MDExtend32Opcode = Js::OpCode::MOVSXD;
 
 extern const IRType RegTypes[RegNumCount];
 
+bool
+LowererMDArch::IsArgSaveRequired(Func *func) {
+    return (!func->IsTrueLeaf() || func->GetHasJitCalls() ||
```

Comment:
>GetHasJitCalls [](start = 41, length = 14)

Should go inside IsTrueLeaf. But if you reuse isLeaf to take care of this, this API itself is not required then.